cosmos/operators/local.py

```diff
@@ -225,6 +219,18 @@ def run_command(
                     self.profile_config.target_name,
                 ]
 
+                if self.install_deps:
```

Comment:
@tatiana We could separate the command creation outside of `run_command`  and bring it into `build_and_run`, then it's easier to test for the arguments

Reply:
It sounds like a good plan, @joppevos !